### PR TITLE
Monarch Pro software integration tutorial

### PR DESCRIPTION
**Pull request by Bitterisland6**
Only software part of tutorial template for Unispectral's Monarch Pro multispectral camera

### CODE REVIEW

**Comment by Krzemien97** [comment]
Content wise it looks fine. I have only few notes:
- I would add is image of the **Monarch Pro** at the beginning of the tutorial (to showcase how it looks like). 
- @Bitterisland6 can you add some images to the section in which you mention `rqt dynamic reconfigure`? 

@bjsowa can you check the tutorial from the software side?

**Review by Krzemien97 — APPROVED** [review]

File: docs/leo-rover/integrations/cameras/unispectral-monarch-pro.mdx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+  figStyle={{
+    width: 600,
+  }}
+/>
+
+## What to expect?
+
+After completing this tutorial, you will have successfully connected a Monarch
+Pro stereo camera to the Leo Rover in ROS, and you will possess basic knowledge
+about the use of multispectral camera.
```
Comment (bjsowa) [inline docs/leo-rover/integrations/cameras/unispectral-monarch-pro.mdx]:
```suggestion
about the use of the multispectral camera.
```

File: docs/leo-rover/integrations/cameras/unispectral-monarch-pro.mdx [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+First you have to download it into `src` directory of your ROS workspace:
+
+```bash
+cd <path_to_your_ros_workspace>/src
+git clone https://github.com/fictionlab/ros_monarch_pro.git
+```
+
+Having the source code, we need have installed all the required dependencies so
+the package will be installed successfully. To install the dependencies type:
```
Comment (bjsowa) [inline docs/leo-rover/integrations/cameras/unispectral-monarch-pro.mdx]:
```suggestion
Having the source code, we need to install all of the required dependencies so
the package can be built successfully. To install the dependencies, type:
```

File: docs/leo-rover/integrations/cameras/unispectral-monarch-pro.mdx [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+```bash
+source devel/setup.bash
+rosdep install --from-paths src -iry
```
Comment (bjsowa) [inline docs/leo-rover/integrations/cameras/unispectral-monarch-pro.mdx]: It's more failsafe to always update the package lists and rosdep database before running `rosdep install`
```suggestion
sudo apt update
rosdep update
rosdep install --from-paths src -iry
```

Comment (bjsowa) [reply docs/leo-rover/integrations/cameras/unispectral-monarch-pro.mdx]: also, there is no `devel` directory if the workspace was never built previously

File: docs/leo-rover/integrations/cameras/unispectral-monarch-pro.mdx [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+Before starting any of the programs, make sure you completed the **Connecting
+another computer to ROS network** of **ROS development** tutorial:
+
+<LinkButton docId="leo-rover/advanced-guides/ros-development" />
+
+Now you can start `rqt`, and from `plugins` -> `configuration` choose
+`dynamic reconfigure` and then choose the `monarch_pro_wrapper` node. On the
+right side of the screen you will see the mentioned parameters - you can use the
+sliders and boxes next to them to change the values of the parameters.
+
```
Comment (bjsowa) [inline docs/leo-rover/integrations/cameras/unispectral-monarch-pro.mdx:163]: Add a section about adding the node to autostart in `/etc/ros/robot.launch`

File: docs/leo-rover/integrations/cameras/unispectral-monarch-pro.mdx [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+We have prepared special ROS wrapper for the unispectral's Monarch Pro camera.
+
+### Installing the packages
+
+First you have to download it into `src` directory of your ROS workspace:
+
+```bash
+cd <path_to_your_ros_workspace>/src
+git clone https://github.com/fictionlab/ros_monarch_pro.git
```
Comment (bjsowa) [inline docs/leo-rover/integrations/cameras/unispectral-monarch-pro.mdx:85]: What if the user does not have a directory for a local ROS workspace?

File: docs/leo-rover/integrations/cameras/unispectral-monarch-pro.mdx [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+### Configuring the node
+
+In the downloaded repository, there is a config file with camera parameters. The
+parameters are loaded to the node automatically during the launch process. You
+can change the provided camera parameters to your desired values (available
+settings are `gain` and `exposure fps`). To load them into the program and your
+camera, you need to restart the node.
+
+If you have ROS installed on your computer, you can also change the camera image
+with rqt plugin - dynamic reconfigure - which doesn't require node restart.
```
Comment (bjsowa) [inline docs/leo-rover/integrations/cameras/unispectral-monarch-pro.mdx]:
```suggestion
If you have ROS installed on your computer, you can also change the camera wrapper parameters in runtime using the rqt dynamic reconfigure plugin.
```

**Review by bjsowa — CHANGES_REQUESTED** [review]

File: docs/leo-rover/integrations/cameras/unispectral-monarch-pro.mdx (hunk 1 above)
Comment (Bitterisland6) [reply docs/leo-rover/integrations/cameras/unispectral-monarch-pro.mdx]: Done.

File: docs/leo-rover/integrations/cameras/unispectral-monarch-pro.mdx (hunk 2 above)
Comment (Bitterisland6) [reply docs/leo-rover/integrations/cameras/unispectral-monarch-pro.mdx]: Done.

File: docs/leo-rover/integrations/cameras/unispectral-monarch-pro.mdx (hunk 3 above)
Comment (Bitterisland6) [reply docs/leo-rover/integrations/cameras/unispectral-monarch-pro.mdx]: Done - applied suggested changes and added info about the need of local ros workspace.

File: docs/leo-rover/integrations/cameras/unispectral-monarch-pro.mdx (hunk 4 above)
Comment (Bitterisland6) [reply docs/leo-rover/integrations/cameras/unispectral-monarch-pro.mdx:163]: I think it's redundant right now.

File: docs/leo-rover/integrations/cameras/unispectral-monarch-pro.mdx (hunk 5 above)
Comment (Bitterisland6) [reply docs/leo-rover/integrations/cameras/unispectral-monarch-pro.mdx:85]: Fair point - added a note about the need of having one and a link to our tutorial on how to create one.

File: docs/leo-rover/integrations/cameras/unispectral-monarch-pro.mdx (hunk 6 above)
Comment (Bitterisland6) [reply docs/leo-rover/integrations/cameras/unispectral-monarch-pro.mdx]: Done.

**Review by bjsowa — APPROVED** [review]

**Pull request merged**